svf/lib/SABER/SaberSVFGBuilder.cpp

```diff
@@ -110,6 +111,21 @@ PointsTo& SaberSVFGBuilder::CollectPtsChain(BVDataPTAImpl* pta,NodeID id, NodeTo
     else
     {
         PointsTo& pts = cachedPtsMap[baseId];
+        // base object
+        if (Options::IgnoreExtFun())
+        {
+            if (pta->isFIObjNode(baseId))
+            {
+                PAGNode* pn = pag->getGNode(baseId);
+                const SVFValue* value = pn->getValue();
```

Comment:
pn may not have a value, it could be a dummy node.

if(pta->isFIObjNode(baseId) && pag->getGNode(baseId)->hasValue())
{
   const SVFCallInst* inst = SVFUtil::dyn_cast<SVFCallInst>( pag->getGNode(baseId)->getValue());
   if(inst ...)
   ....

}